components/x-audio/src/components/styles.scss

```diff
@@ -359,3 +358,36 @@ $primary-scrubber-color: oColorsGetPaletteColor('teal');
 // 	width: var(--buffered-width);
 // 	background-color: green;
 // }
+
+@media (orientation: landscape) and (max-height: 620px - 1) {
```

Comment:
Just looking at this, I suppose the actual issue here is just the `max-height`, not necessarily the orientation. Is that right? Like if you shrank your browser down to a small height, you'd probably also want these rules to apply. What do you think?

Reply:
@benbarnett That's true. I'm going to remove the landscape query. Thank you!